src/pages/index.js

```diff
@@ -53,24 +21,17 @@ const api = new Api({
 api
   .getAppInfo()
   .then(([cards, userInfo]) => {
+    profileAvatar.src = userInfo.avatar;
     document.querySelector(".profile__avatar").src = userInfo.avatar;
```

Comment:
`document.querySelector(".profile__avatar").src = userInfo.avatar;` is not needed because you have `profileAvatar.src = userInfo.avatar;` for that